lib/screens/home_screen.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:flutter/material.dart';
+import 'package:qlink/screens/screens.dart';
+
+class HomePage extends StatefulWidget {
+  HomePage({Key? key, required this.title}) : super(key: key);
+
+  final String title;
+
+  @override
+  _HomePageState createState() => _HomePageState();
+}
+
+class _HomePageState extends State<HomePage> {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
```

Comment:
So we totally don't have to do it for this project, but one ambition I had last year was looking at how to create a Flutter app that looked native to Android or iOS depending on which platform you were on.

So the apps would actually look different, but the code would be written a single way, and depending on the platform, either the Material or the Cupertino option was used.

I had wanted a gimme library for that and found the ecosytem lacking at the time.

MVP probably not worth it, but it's one of those things that's hard to refactor for later.

Maybe this can be one of my first things I look into since I'm the annoying, picky iOS user 😉 .